spec/integration/puppet_x/puppetlabs/powershell_manager_spec.rb

```diff
@@ -31,8 +31,9 @@ module WindowsAPI
 end
 
 describe PuppetX::PowerShell::PowerShellManager,
-  :if => Puppet::Util::Platform.windows? && PuppetX::PowerShell::PowerShellManager.supported? do
-
+  :if => Puppet::Util::Platform.windows? && PuppetX::PowerShell::PowerShellManager.supported?,
```

Comment:
This appears to have been because it wasn't well understood that the pipe race fix in #3144 should have been performed first.  After that merge, the only failing test (aside from the ones fixed in this PR) is the 64k test.  The only reason that test was failing was due to the difference in behavior between `Write-Output` in PS2 and PS3+ (PS2 introduces many extra newlines).

* #149 has been merged to address this concern.  That re-enabled 48 of the 49 tests.
* #150 reworked the tests so that all tests can be run regardless of PowerShell version

So at this point, we're good!